app/views/devise/registrations/edit.html.haml

```diff
@@ -6,9 +6,17 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
     = devise_error_messages!
```

Comment:
That's not what I am seeing. The errors on the membership application page seem to render correctly without the added breaks.